solr.sh

```diff
@@ -23,4 +23,4 @@ cd -
 # Start the server.
 cd solr/example/
 
-java -jar start.jar
+java -jar start.jar > /dev/null 2>&1 < /dev/null &
```

Comment:
We would need to provide an argument for that. And also update the readme file.

10x.